firefox-ios/Shared/SupportUtils.swift

```diff
@@ -37,6 +37,24 @@ public struct SupportUtils {
         return URL(string: "https://support.mozilla.org/1/mobile/\(AppInfo.appVersion)/iOS/\(languageIdentifier)/\(escapedTopic)")
     }
 
+    public static func URLForPrivacyNotice(source: String, campaign: String, content: String?) -> URL? {
```

Comment:
originally tried to work with the review checker, but the requirements were a bit different so this only applies to microsurvey currently. However, it seems that there are certain requirements with marketing in the utm params so going forward, we can use this to maintain consistency in our privacy notice URLs and ensure localization. cc: @thatswinnie

Reply:
Also debated on making the source and campaign enums, but since its only use in one place for now, I left as string. We can update to enum in the future. Let me know if any issues!